calib/README.md

```diff
@@ -0,0 +1 @@
+Directory for (master) calibs
```

Comment:
There are no master calibs (AFAIK) in obs_test, so not clear where to get one.
But I realized that I forgot to edit the main README so I will do that now.